config/robotframework_aio/release_items_RobotFramework_UDS.json

```diff
@@ -97,6 +97,12 @@ Example test case with UDS:
 
 Please refer to `RobotFramework_UDS documentation <https://github.com/test-fullautomation/robotframework-uds/blob/develop/RobotFramework_UDS/RobotFramework_UDS.pdf>`_
 for more details.
+"
+                                       ],
+                   "0.13.1." : [
+"
+* Updated the UDS keywords to support working with multiple connections.

```

Comment:
Hi Tri,
can you please also add the "please refer to ...." from above.
Are we still compatible to already existing test cases? As long as there is only one connection `device` is not required. Did you try running old tests in order to assure that there are no effects?
Thank you,
 Thomas

Reply:
Hi @test-fullautomation,
I will add the "please refer to..." as requested.
Regarding compatibility with the existing test cases:  I have run the old tests to ensure that there are no side effects, and everything is functioning as expected.
Thank you, 
Tri

Reply:
very good!

Reply:
Hi @trimai3001 ,
is the PDF-documentation updated? I see only a robot file with examples how to use.
PDF documentation is missing?
Thank you,
 Thomas